app/core/error_handlers.py

```
@@ -0,0 +1,36 @@
+from fastapi import Request, HTTPException
+from fastapi.responses import JSONResponse
+from fastapi.exceptions import RequestValidationError
+from loguru import logger
+import sqlalchemy.exc
+
+def format_error_response(status_code: int, detail: str, error_type: str = "error"):
+    return {"status": "error", "error_type": error_type, "detail": detail}
+
+async def http_exception_handler(request: Request, exc: HTTPException):
```

Ruff (0.8.2)

Missing return type annotation for public function `http_exception_handler`

(ANN201)